cllc-public-app/Controllers/ApplicationsController.cs

```diff
@@ -1115,10 +1115,21 @@ public async Task<IActionResult> UpdateApplication([FromBody] ViewModels.Applica
                         AdoxioSaturdayclose = (int?)item.ServiceHoursSaturdayClose,
                         AdoxioSaturdayopen = (int?)item.ServiceHoursSaturdayOpen
                     };
-                    
-                    _dynamicsClient.Hoursofservices.Update(hoursEntity.AdoxioHoursofserviceid, patchHoursEntity);
+
+                    if (hoursEntity != null)
+                    {
+                        _dynamicsClient.Hoursofservices.Update(hoursEntity.AdoxioHoursofserviceid, patchHoursEntity);
```

Comment:
This update should have a try-catch (every call to dynamics is a possible point of failure)